Formula/gammalib.rb

```diff
@@ -0,0 +1,41 @@
+class Gammalib < Formula
+  desc "Toolbox for the analysis of astronomical gamma-ray data"
+  homepage "http://cta.irap.omp.eu/gammalib/about.html"
+  url "http://cta.irap.omp.eu/ctools/releases/gammalib/gammalib-1.6.3.tar.gz"
+  sha256 "b93bc03562ba1fcb21ab6f48943f9c28230c04d1b0ef986569c88b05d8e9dbbe"
+  revision 1
+
+  depends_on "cfitsio"
+  depends_on "doxygen"
+  depends_on "ncurses"
+  depends_on "python"
```

Comment:
Please try with python@3.8

Reply:
checking whether /usr/local/bin/python3 version is >= 2.3... no
I test on my Mac, it's OK with python or python3.8, 
I don't know why.
could you help?

Reply:
I'm not sure what you mean.

Reply:
I build it OK with depends_on python, But with some error with python@3.8, should I use python@3.8 by force? or it's OK with the default python3.7